modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModalImportObjectDefinition.tsx

```diff
@@ -261,7 +270,7 @@ export default function ModalImportObjectDefinition({
 										);
```

Comment:
@Artartur one more thing, here we have to clear the error here, for example, if the user imports for the first time a structure that fails in the import process, we will show the error, but if after that he imports a valid structure the error will remain on the screen causing a wrong feeling then we have to clear this error after successful attempt:

```tsx
setError('');
```

Reply:
I'm not sure if I've added it in the right place, can you confirm?

Reply:
@Artartur actually you have to put this `setError('');` here, inside this `try` statement after `setExternalReferenceCode`

Reply:
Done!